### PR TITLE
Added missing jQuery.Event properties

### PR DESCRIPTION
**Pull request by vobruba-martin**

### CODE REVIEW

**Comment by vobruba-martin** [comment]
I signed it!

File: contrib/externs/jquery-1.9.js [hunk 1]
```diff
@@ -645,11 +651,20 @@ jQuery.Event.prototype.isImmediatePropagationStopped = function() {};
 jQuery.Event.prototype.isPropagationStopped = function() {};
 
 /** @type {boolean} */
+jQuery.Event.prototype.isTrigger;
```
Comment (ChadKillingsworth) [inline contrib/externs/jquery-1.9.js]: I don't believe this is a documented property of jQuery events. We only provide externs for documented public properties.

File: contrib/externs/jquery-1.9.js [hunk 2]
```diff
@@ -645,11 +651,20 @@ jQuery.Event.prototype.isImmediatePropagationStopped = function() {};
...
+jQuery.Event.prototype.keyCode;
+
+/** @type {boolean} */
 jQuery.Event.prototype.metaKey;
 
 /** @type {string} */
 jQuery.Event.prototype.namespace;
 
+/** @type {RegExp} */
+jQuery.Event.prototype.namespace_re;
```
Comment (ChadKillingsworth) [inline contrib/externs/jquery-1.9.js]: This is another non-documented property. If you feel like it should be included, then perhaps a PR to api.jquery.com to document the property first is in order. Feel free to cross link to this PR.

File: contrib/externs/jquery-1.9.js [hunk 3]
```diff
@@ -701,6 +716,9 @@ jQuery.Event.prototype.target;
 /** @type {number} */
 jQuery.Event.prototype.timeStamp;
 
+/** @type {?} */
+jQuery.Event.prototype.toElement;
```
Comment (ChadKillingsworth) [inline contrib/externs/jquery-1.9.js]: Shouldn't the type here be `Element`?

File: contrib/externs/jquery-1.9.js [hunk 4]
```diff
@@ -626,6 +629,9 @@ jQuery.Event.prototype.detail;
 /** @type {number} */
 jQuery.Event.prototype.eventPhase;
 
+/** @type {Object} */
+jQuery.Event.prototype.handleObj;
```
Comment (ChadKillingsworth) [inline contrib/externs/jquery-1.9.js]: This is another undocumented property.

**Comment by ChadKillingsworth** [comment]
Thanks for the PR! I've left a few comments in regards to undocumented properties.

**Comment by vobruba-martin** [comment]
I've updated the PR. Thanks for your comments!

**Pull request merged**